### PR TITLE
update readme to reflect data options

### PR DESCRIPTION
**Pull request by jon-stevens**
Small update to demo to illustrate using static data rather than using an endpoint to retrieve results

### CODE REVIEW

File: packages/global-autocomplete/demo/main.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ const args = {
 	selector: '[data-component-autocomplete]',
 	onSelect: onSelect,
 	searchError: onError,
-	endpoint: 'autocomplete?q=',
+	endpoint: 'autocomplete?q=', // OPTIONAL: Provide either endpoint or array of data - if both are provided static data will be used
+	//staticResultsData: animalsList, // OPTIONAL: Provide either endpoint or array of data - if both are provided array be used
```
Comment (davidpauljunior) [inline packages/global-autocomplete/demo/main.js]: Is the `//staticResultsData` part meant to be commented?

Comment (jon-stevens) [reply packages/global-autocomplete/demo/main.js]: It's just to illustrate that you don't need to pass in both an endpoint and the static data.

I did want to pass in staticResultsData and comment out endPoint instead but I didn't want to potentially break Jon's demo

Comment (davidpauljunior) [reply packages/global-autocomplete/demo/main.js]: Oh I get ya, sorry.

File: packages/global-autocomplete/HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 1.0.2 (2019-10-16)
+    * Update readme
```
Comment (jpw) [inline packages/global-autocomplete/HISTORY.md]: The date is wrong, and this PR does not update the readme? :)

File: packages/global-autocomplete/HISTORY.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 1.0.3 (2020-01-06)
```
Comment (alexkilgour) [inline packages/global-autocomplete/HISTORY.md]: also update package.json file to new version?

**Pull request merged**